doc/neps/nep-0022-ndarray-duck-typing-overview.rst

```diff
@@ -138,7 +138,7 @@ person even understands all the distinctions. And this is important,
 because we have a *lot* of APIs that we need to add duck array support
 to (both in numpy and in all the projects that depend on numpy!). By
 definition, these already work for ``ndarray``, so hopefully getting
-them to work for full duck arrays shouldn’t be so hard, since by
+them to work for full duck arrays shouldn't be so hard, since by
```

Comment:
This should be left alone.

Reply:
not per se a typo fix replacing U+2019 with U+0027 but pursued as it is easier to deal with dumped strings

if you still think it should be left alone it can easily be reverted

Reply:
Yes, please leave it alone. We use that style of quote in the rest of the file.